elliott/elliottlib/cli/find_bugs_qe_cli.py

```diff
@@ -68,6 +69,15 @@ def find_bugs_qe(runtime, find_bugs_obj, noop, bug_tracker):
     """
                 bug_tracker.add_comment(bug.id, comment, private=True, noop=noop)
 
+                # get summary of CVE bug and update it if needed
+                summary_suffix = f"[openshift-{major_version}.{minor_version}]"
+                if not bug.summary.endswith(summary_suffix):
+                    new_s = fix_summary_suffix(bug.summary, summary_suffix)
+                    try:
+                        bug.update_summary(new_s)
```

Comment:
let's respect dry-run here, so we can pass noop param and then in update_summary , if noop is true, then only log and return

Reply:
Done